Musical Collection/Musical Collection Console App/Classes/Listener.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Musical_Collection_Console_App.Interfaces.Classes_Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Musical_Collection_Console_App.Classes
+{
+    public class Listener : User, IListener 
+    {
+        private List<string> _favouriteGenres;
```

Comment:
In all your private collection you are using List change it to IEnumarble and
assain it to List in the ctor